### PR TITLE
RI-376 Add Queens branch gating

### PR DESCRIPTION
**Pull request by cjloader**
Adds gate jobs for the new branches

Issue: [RI-376](https://rpc-openstack.atlassian.net/browse/RI-376)

### CODE REVIEW

File: rpc_jobs/rpc_openstack.yml [hunk 1]
```diff
@@ -48,6 +48,52 @@
       - "tox-test"
     jobs:
       - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
+- project:
```
Comment (mattt416) [inline rpc_jobs/rpc_openstack.yml]: Nit: space about `- project:`

File: rpc_jobs/rpc_openstack.yml [hunk 2]
```diff
@@ -246,6 +292,110 @@
...
+       - deploy
+     exclude:
+       - action: system
+         scenario: ironic
+       - action: sdqc
+         scenario: ironic
+     # Required by RPC-ASC team to upload test results qTest
+     credentials: "rpc_asc_creds"
+     jobs:
+       - 'PM_{repo_name}-{branch}-{image}-{scenario}-{action}'
```
Comment (mattt416) [inline rpc_jobs/rpc_openstack.yml]: Space below pls! :)

File: rpc_jobs/rpc_openstack.yml [hunk 3]
```diff
@@ -465,6 +615,24 @@
...
+          FLAVOR: "onmetal-io2"
+          IMAGE: "OnMetal - Ubuntu 16.04 LTS (Xenial Xerus)"
+          REGIONS: "IAD"
+          FALLBACK_REGIONS: "DFW"
+    scenario:
+      - swift
+    action:
+      - deploy
+    jobs:
+      - 'RELEASE_{repo_name}-{branch}-{image}-{scenario}-{action}'
```
Comment (mattt416) [inline rpc_jobs/rpc_openstack.yml]: Space below pls! :)

File: rpc_jobs/rpc_openstack.yml [hunk 4]
```diff
@@ -246,6 +292,110 @@
...
+      - xenial_mnaio_no_artifacts:
+          FLAVOR: "onmetal-io2"
+          IMAGE: "OnMetal - Ubuntu 16.04 LTS (Xenial Xerus)"
+          REGIONS: "IAD"
+          FALLBACK_REGIONS: "DFW"
+    scenario:
+      - ironic
+      - swift
+    action:
+      - system
```
Comment (mattt416) [inline rpc_jobs/rpc_openstack.yml]: @johnduarte you all OK w/ these jobs appearing?

Comment (mattt416) [reply rpc_jobs/rpc_openstack.yml]: Got feedback that ASC is happy with these appearing.

File: rpc_jobs/rpc_openstack.yml [hunk 5]
```diff
@@ -49,6 +49,53 @@
     jobs:
       - 'PR_{repo_name}-{series}-{image}-{scenario}-{action}'
 
+- project:
+     name: "rpc-openstack-queens-premerge"
```
Comment (mattt416) [inline rpc_jobs/rpc_openstack.yml]: The indentation of all these projects appears to be off. I think this is the reason for the failure.

File: rpc_jobs/rpc_openstack.yml (hunk 4 above)
Comment (johnduarte) [reply rpc_jobs/rpc_openstack.yml]: 👍

**Pull request merged**